src/widget/core.rs

```diff
@@ -130,6 +132,9 @@ impl WidgetState {
 
     fn merge_up(&mut self, child_state: &mut WidgetState) {
         self.flags |= child_state.upwards_flags();
+        if child_state.flags.contains(PodFlags::REQUEST_ACCESSIBILITY) {
+            self.flags |= PodFlags::DESCENDANT_REQUESTED_ACCESSIBILITY;
+        }
```

Comment:
I think we should do this in `WidgetState::upwards_flags()` to avoid duplication.